linker-interface/shared/src/test/scala/org/scalajs/linker/interface/StandardConfigFingerprintTest.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.linker.interface
+
+import java.net.URI
+
+import org.junit.Test
+import org.junit.Assert._
+
+import Semantics.RuntimeClassNameMapper._
+
+class StandardConfigFingerprintTest {
+  @Test
+  def noFingerprintCollisionCheckIR(): Unit = {
+    val sc1 = StandardConfig().withCheckIR(false)
+    val sc2 = StandardConfig().withCheckIR(true)
+    assertNotEquals(
+      StandardConfig.fingerprint(sc1),
+      StandardConfig.fingerprint(sc2)
+    )
```

Comment:
Consider a helper for this. Also indent 4 otherwise.

Reply:
Fixed in eb6e308689224c3bbcc9c059e10fa1f02bee3acb